File: 백준/Bronze/9063. 대지/대지.js
let fs = require('fs')
let input = fs.readFileSync("/dev/stdin").toString().trim().split("\n")
// let input = fs.readFileSync("./EX.txt").toString().trim().split("\n")

const N = parseInt(input[0])

xMax = -Infinity
xMin = Infinity
yMax = -Infinity
yMin = Infinity

if (N !== 1) {
    for (let i = 1; i <= N; i++) {
        const line = input[i].split(" ")
        const a = parseInt(line[0])
        const b = parseInt(line[1])

        xMax = Math.max(xMax, a)
        xMin = Math.min(xMin, a)
        yMax = Math.max(yMax, b)
        yMin = Math.min(yMin, b)
    }

    console.log((xMax - xMin) * (yMax - yMin))
} else {
    console.log(0)
}

File: 백준/Bronze/9063. 대지/대지.py
import sys
input = sys.stdin.readline

N = int(input())
x = []
y = []
if N > 1:
    for _ in range(N):
        a, b = map(int, input().split())
        x.append(a)
        y.append(b)

    result = (max(x) - min(x)) * (max(y) - min(y))
    print(result)

else:
    print(0)